packages/hash/frontend/tsconfig.json

```diff
@@ -6,7 +6,12 @@
     "jsx": "preserve",
     "lib": ["dom", "dom.iterable", "esnext"],
     "target": "es5",
-    "noEmit": true
+    "noEmit": true,
+    "paths": {
+      "@blockprotocol/type-system": [
+        "../../../../node_modules/@blockprotocol/type-system/dist/cjs-slim/index-slim.d.ts"
+      ]
```

Comment:
I also didn't need to do it in [the example template](https://github.com/Alfred-Mountfield/wasm-sandbox/blob/main/nextjs-app/tsconfig.json) so I'm not sure why we need this here.

Deleting it doesn't seem to complain in my IDE (perhaps you're using a feature of the IDE I'm not?), maybe we should jump on a call and have a look?

Reply:
Discussed on a call that this is so typescript knows to use the slim package's typings instead of the non-slim typings (which happen to be the same here, but this is for completeness/future proofing)